### PR TITLE
more typos fixed

### PR DESCRIPTION
**Pull request by stenflo**
I found a few more typos: coordinatates -> coordinates, geodecentric -> geocentric, contributers -> contributors,
inital -> initial

The spelling correction does affect the following fields in the response object:

```
observer.longitudeGeodecentric
observer.latitudeGeodecentric
observer.heightGeodecentric
```

I would say these could be considered non-breaking changes because the exported function signatures haven't changed, just these fields in the response object, in my subjective opinion ;-)

The pull request includes a few separate commits, and includes some code formatting also...
Most of the code uses 2 spaces for indentation, but `transit.js` was using mostly tab characters, so I replaced those
with spaces for consistency.

also, made a small change to the build script. the build artifact doesn't need to be included in the repo, so I removed `'build/index.js'` and added the 'build' directory to `'.gitignore'` for exclusion.

Please take your time to review. I think the typo corrections are important to promote understanding of these obscure concepts as well as confidence in the api. Hopefully this is the last of them. I would suggest either release this as 2.1 or hold it before you make a 3.0 version as there could be a few more things to include before another major version.

What's next? I'm working on a types package. The way it works is there can be a separate npm package:
```
npm install --save ephemeris
npm install --save-dev @types/ephemeris
```
Nothing needs to change with the ephemeris project, the separate npm package is optional and would provide type definitions for ephemeris when used with TypeScript. To create the '@types/ephemeris' package I would be making a pull request to the @DefinitelyTyped repo which is responsible for generating the `@types` npm package.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "planets"
   ],
   "author": "Hemantkumar Goswami",
-  "license": "GPL-3.0",
+  "license": "GPL-2.0",
```
Comment (hemantgoswami) [inline package.json]: Why would you want to change the version of GPL?

**Review by hemantgoswami — CHANGES_REQUESTED** [review]
Rest everything looks good only question raised is about the downgrade of GPL licence version. Any specific reason for that?

File: package.json (hunk 1 above)
Comment (stenflo) [reply package.json]: it was not my intention to _change_ the license type, only to make all references _consistent_. The license type is referenced in 3 different files: `README.md`, `package.json`, and the `LICENSE` file itself. The README and the LICENSE both say GPL 2.0 but `package.json` says GPL 3.0. Also, I can see in the commit history that you "downgraded" the license type from GPL 3.0 to GPL 2.0 on 03/05/2018, so I presumed GPL 2.0 is the desired license type. Therefore I changed `package.json` to match what is already in `LICENSE` and the `README.md`

Comment (hemantgoswami) [reply package.json]: I see, its my bad then, do you want to make all of it 3.0 please?

**Review by hemantgoswami — APPROVED** [review]
LGTM

**Pull request merged**